app/tests/algorithms_tests/test_views.py

```diff
@@ -1367,3 +1369,20 @@ def test_algorithm_model_version_management(settings, client):
     assert not m2.is_desired_version
     del alg.active_model
     assert not alg.active_model
+
+@pytest.mark.django_db
+def test_job_list_row_template_renders(
+    client, settings, django_capture_on_commit_callbacks
+):
+    job = AlgorithmJobFactory()
+
+    assert job.algorithm_image
+
+    job.status = Job.SUCCESS
```

Comment:
You can add the status directly to the factory. Also, in most cases, you will need to save the object again after modifying it. In this case, it doesn't matter, but in most other cases it does.

Reply:
Sure, I will keep that in mind. Should I pass the status to the factory and save the object for this PR?